fastlane/lib/fastlane/actions/slather.rb

```diff
@@ -287,13 +271,14 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :source_files,
                                        env_name: "FL_SLATHER_SOURCE_FILES",
                                        description: "A Dir.glob compatible pattern used to limit the lookup to specific source files. Ignored in gcov mode",
-                                       is_string: false,
+                                       type: Array,
+                                       skip_type_validation: true, # skipping validation for backwards compatibility with Boolean type
```

Comment:
Ummm... does `type` do anything when `skip_type_validation` is true? 🤔 I actually don't know lol

Reply:
I _think_ we maybe need to remove `type` but 🤷‍♂️

Reply:
> Ummm... does type do anything when skip_type_validation is true? 🤔 I actually don't know lol

Nice question, **type doesn't do anything** when skip_type_validation is true because we skip all the type checks straight-away, see: 
https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L200 

> I think we maybe need to remove type

Yes, I will remove it, anyways type will get ignore always 😉